generator/src/main.rs

```diff
@@ -1379,6 +1379,8 @@ impl Parser {
                 }
             })
             .collect::<IndexSet<&str>>();
+
+        let forced_reexports = ["UuidEXT"];
```

Comment:
Rather than hardcoding static stuff into the generator, it's easier and more maintainable to add it to the appropriate lib.rs directly.